backend/src/app.ts

```diff
@@ -1,4 +1,6 @@
 import app from "./server";
+import serverless from "serverless-http";
```

Comment:
`serverless-http` is not listed in `package.json` and it looks like `serverless` is not being used

Reply:
yep forgot to remove it. was testing some deployment stuff. thanks. will update